mm2src/lp_ordermatch.rs

```diff
@@ -356,8 +386,26 @@ pub async fn process_msg(ctx: MmArc, _topics: Vec<String>, from_peer: String, ms
             }
             match message {
                 new_protocol::OrdermatchMessage::MakerOrderCreated(created_msg) => {
-                    let order: OrderbookItem = (created_msg, hex::encode(pubkey.to_bytes().as_slice())).into();
+                    let pubkey_str = hex::encode(pubkey.to_bytes().as_slice());
+                    let order: OrderbookItem = (created_msg.clone(), pubkey_str.clone()).into();
                     insert_or_update_order(&ctx, order).await;
+                    if let Some(protocol_info) = created_msg.base_protocol_info {
```

Comment:
Is this intended for logging purposes only? Please remove, if yes. Also logging every created order to the `INFO` level may overflow the logs quite fast, please change the level to `debug` if you prefer to keep the log.

Reply:
When we discussed this issue I thought that the comment regarding that protocol info should be also used on MakerOrderCreated processing to display the order creator's address in a proper format was here but now I see that you meant this for the orderbook 🙂, will remove this.